nextcloud-startup-script.sh

```diff
@@ -246,7 +246,6 @@ download_script STATIC setup_secure_permissions_nextcloud
 download_script STATIC change_db_pass
 download_script STATIC nextcloud
 download_script NETWORK update-config
-download_script NETWORK trusted
 download_script MENU menu
 download_script MENU server_configuration
 download_script MENU nextcloud_configuration
```

Comment:
Could we somehow work around the initial downloading? then we wouldn't have to track which script is already downloaded (and execute the downloaded one) but instead always run the newest script from the repo...
(which should theoratically be no problem since when the user is at this point the internet of the VM is working)

Reply:
That would make the startup script more prune to errors. Is that what we want?

Reply:
Probably not...